readinglist/__init__.py

```diff
@@ -97,9 +108,36 @@ def on_new_response(event):
     config.add_subscriber(on_new_response, NewResponse)
 
 
+def end_of_life_tween_factory(handler, registry):
+    """Pyramid tween to handle service end of life."""
+
+    def eos_tween(request):
+        eos_date = registry.settings.get("readinglist.eos")
+        eos_url = registry.settings.get("readinglist.eos_url")
+        if eos_date:
+            eos_date = dateparser.parse(eos_date)
```

Comment:
Could this go out of the tween (using the closure) to prevent parsing setting at each request ?